Gemfile.lock

```diff
@@ -2,6 +2,7 @@ GEM
   remote: http://rubygems.org/
   specs:
     RedCloth (4.2.9)
+    RedCloth (4.2.9-x86-mingw32)
```

Comment:
I mean this file shouldn't have the changes checked in. I don't know what adding those versions will do for everyone else including a build system.